src/server.rs

```diff
@@ -173,7 +173,7 @@ where
 
 #[instrument(level="info", name="api_request", skip(app), fields(http.uri=%request.uri(), http.method=%request.method()))]
 async fn route(request: Request<Body>, app: Arc<App>) -> Result<Response<Body>, hyper::Error> {
-    trace_from_headers(request.headers());
+    // trace_from_headers(request.headers());
```

Comment:
This is orthogonal to eyre, but makes sense to remove since it also touches OTLP/tracing and could cause trouble.